Misc/NEWS.d/next/Documentation/2020-11-26-21-45-09.bpo-42451.i4QoA_.rst

```diff
@@ -0,0 +1,2 @@
+update PyTuple_GetItem docs to indicate that it does not support negative
```

Comment:
I don't think that it's worth it, you can remove it ;-) I add the skip news label.